pipelines/internal/common/common.go

```diff
@@ -51,3 +55,29 @@ func (m *MapFlagValue) Set(input string) error {
 	}
 	return nil
 }
+
+// PipelineExecutionError is an error returned by the Genomics API
```

Comment:
This is OK for now: we may end up changing this to a generic APIError as we build more API specific behaviours.